flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -103,6 +106,23 @@ static TableEnvironment create(EnvironmentSettings settings) {
 	 */
 	Optional<Catalog> getCatalog(String catalogName);
 
+	/**
+	 * Load a module with given name.
```

Comment:
Nit: use a similar comment as those in existing methods.

Reply:
Please extend the javadoc with description what is a `Module`